### PR TITLE
Show envelope meta data and collapse accordions by default

### PR DESCRIPTION
**Pull request by daniellemaxwell**
### Scope of changes

Displays envelope meta data inside an accordion component that is closed by default. This PR also includes a small update to a couple commands in the `README`.

### Type of change

- [ ] bug fix
- [x] new feature
- [ ] documentation
- [ ] other (describe)

### Acceptance criteria

https://www.awesomescreenshot.com/video/28584964?key=aba6fcec3d44b2860b43875c14a20350

### Author checklist

- [x] I have manually tested the change and/or added automation in the form of unit tests or integration tests
- [ ] I have updated the dependencies list
- [ ] I have added new test fixtures as needed to support added tests
- [ ] I have added or updated the documentation
- [x] Check this box if a reviewer can merge this pull request after approval (leave it unchecked if you want to do it yourself)

### Reviewer(s) checklist

- [ ] Any new user-facing content that has been added for this PR has been QA'ed to ensure correct grammar, spelling, and understandability.
- [ ] To the best of my ability, I believe that this PR represents a good solution to the specified problem and that it should be merged into the main code base.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,15 +65,15 @@ Note this requires you to have [Go](https://go.dev/doc/install) installed on you
 Step 5: Create an admin user to login to the localhost with
 
 ```
-$ docker compose exec envoy envoy createuser -e [email] -r admin
+$ docker compose exec envoy.local envoy createuser -e [email] -r admin
```
Comment (daniellemaxwell) [inline README.md:68]: Updated to reflect a change in the service name.

File: README.md [hunk 2]
```diff
@@ -65,15 +65,15 @@ Note this requires you to have [Go](https://go.dev/doc/install) installed on you
...
 ```
 
 Now open a browser at [http://localhost:8000](http://localhost:8000) (or at [http://envoy.local:8000](http://envoy.local:8000) if you have edited your hosts file) and you should be able to access the envoy node with the email and password created in the previous step!
 
 Step 6: Optionally create an admin user to login to the local counterparty with. The counterparty is intended to allow you to have two Envoy nodes to send transfers back and forth to.
 
 ```
-$ docker compose exec counterparty envoy createuser -e [email] -r admin
+$ docker compose exec counterparty.local envoy createuser -e [email] -r admin
```
Comment (daniellemaxwell) [inline README.md:76]: Same as above.

File: pkg/web/static/js/transactionInfo.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+          });
+        }
+      });
+    });
+
+    // TODO: Add code to toggle show/hide for PKS.
+  }
+});
+
+// Add code to amend the request parameters before the request is sent.
```
Comment (daniellemaxwell) [inline pkg/web/static/js/transactionInfo.js:28]: The code starting here isn't new. It moved from the partial to the `js` file.

**Review by pdamodaran — APPROVED** [review]
Nicely done @daniellemaxwell - I like the accordion feature with the tooltip!  Thanks for updating the README!

**Pull request merged**